client/src/components/Channel.js

```diff
@@ -168,9 +168,9 @@ class Channel extends React.Component {
       ChannelActions.sendMessage(this.state.channelName, text);
   }
 
-  sendFile(filePath: string, buffer) {
+  sendFile(filePath: string, buffer, mimeType = 'application/octet-binary') {
```

Comment:
I think it would be nice to have an optional mime type stored in file post metadata (`ipfs-post` would need to support it).